config/pacbio_instrument_types.yml

```diff
@@ -15,10 +16,11 @@ default: &default
               attribute: :plates
               minimum: 1
               maximum: 1
-          required_attributes:
-            options:
-              required_attributes:
-                - :dna_control_complex_box_barcode
+          # TODO: fix this to have separate behaviour for v11 and v12
```

Comment:
Does this still need doing?

Reply:
This is tricky because the validation is a combination of instrument type and smrtlink version. It doesn't really fall under the SmrtLinkOptionsValidator or the InstrumentValidator.